src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -105,7 +106,7 @@ private void testCreateInstructorAction() throws Exception {
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructor.name + " | " + instructor.email + " | " + institute);
         
         String encryptedKey = StringHelper.encrypt(BackDoor.getKeyForInstructor(demoCourseId, instructor.email));
-        String expectedjoinUrl = new teammates.common.util.Url(Const.ActionURIs.INSTRUCTOR_COURSE_JOIN)
+        String expectedjoinUrl = new AppUrl(Const.ActionURIs.INSTRUCTOR_COURSE_JOIN)
```

Comment:
May be add a comment to explain why AppUrl is used here instead of the usual TestUrl?. If not, the reader might not realize it.